### PR TITLE
Create log4j post

### PR DESCRIPTION
**Pull request by rebeccadee**

### CODE REVIEW

File: blogposts/2021/log4j-log4-shell-0-day.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+---
+title: "Log4j Log4Shell 0-day: find, fix, and track affected code with code search"
+description: "How code search helps you deal with log4j CVE-2021-44228:
```
Comment (amieroth) [inline blogposts/2021/log4j-log4-shell-0-day.md:3]: Starting with "How code search helps..." feels self-serving. Can we either delete everything before the colon?

File: blogposts/2021/log4j-log4-shell-0-day.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+find everywhere vulnerable log4j versions are used, automate the creation of pull requests to patch the vulnerability across all your code, and track the progress of the fixes/mitigations."
+author: Quinn Slack
+authorUrl: https://twitter.com/sqs
+publishDate: 2021-12-13T10:00-07:00
+tags: [blog]
+slug: log4j-log4shell-0-day
+heroImage:
+socialImage:
```
Comment (rebeccadee) [inline blogposts/2021/log4j-log4-shell-0-day.md:11]:
```suggestion
socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/log4j/log4j-track-progress-prs.png
```

File: blogposts/2021/log4j-log4-shell-0-day.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
+---
+title: "Log4j Log4Shell 0-day: find, fix, and track affected code with code search"
+description: "How code search helps you deal with log4j CVE-2021-44228:
+find everywhere vulnerable log4j versions are used, automate the creation of pull requests to patch the vulnerability across all your code, and track the progress of the fixes/mitigations."
+author: Quinn Slack
+authorUrl: https://twitter.com/sqs
+publishDate: 2021-12-13T10:00-07:00
+tags: [blog]
+slug: log4j-log4shell-0-day
+heroImage:
```
Comment (rebeccadee) [inline blogposts/2021/log4j-log4-shell-0-day.md:10]: I think we can just leave this blank

File: blogposts/2021/log4j-log4-shell-0-day.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
+---
+title: "Log4j Log4Shell 0-day: find, fix, and track affected code with code search"
+description: "How code search helps you deal with log4j CVE-2021-44228:
+find everywhere vulnerable log4j versions are used, automate the creation of pull requests to patch the vulnerability across all your code, and track the progress of the fixes/mitigations."
```
Comment (rebeccadee) [inline blogposts/2021/log4j-log4-shell-0-day.md:4]:
```suggestion
description: "Find everywhere vulnerable log4j versions are used, automate the creation of pull requests to patch the vulnerability across all your code, and track the progress of the fixes/mitigations."
```

File: blogposts/2021/log4j-log4-shell-0-day.md (hunk 1 above)
Comment (rebeccadee) [reply blogposts/2021/log4j-log4-shell-0-day.md:3]: @amieroth Agreed, I've suggested the change https://github.com/sourcegraph/about/pull/5019/files#r767804373

File: blogposts/2021/log4j-log4-shell-0-day.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+
+## Getting started with Sourcegraph
+
+Free accounts on [Sourcegraph Cloud](https://sourcegraph.com) allow you to search your org private code from GitHub.com and GitLab.com. You can choose which orgs and repositories to sync when signing up or later by visiting Settings > Account > Your repositories.
+
+To use Batch Changes and Code Insights to apply mass fixes and track progress, or if you want to run it on your own laptop or infrastructure, [set up a self-hosted Sourcegraph instance](https://docs.sourcegraph.com/#getting-started). These features are usually part of an enterprise plan, but we're giving out temporary license keys to use these features for log4j-related fixes. Email [log4j-incident-response-help@sourcegraph.com](mailto:log4j-incident-response-help@sourcegraph.com) and we'll reply quickly with a temporary key. For any other questions please email [support@sourcegraph.com](mailto:support@sourcegraph.com).
+
+---
+
+_Thanks to the following people for helping with this post: Olaf Geirsson, Rebecca Dodd, Thorsten Ball, Erica Lindberg, Malo Marrec, Victoria Yunger, Beyang Liu. We welcome [edits to this post](TODO) by pull request._
```
Comment (rebeccadee) [inline blogposts/2021/log4j-log4-shell-0-day.md:102]: TODO

Comment (rebeccadee) [reply blogposts/2021/log4j-log4-shell-0-day.md:102]:
```suggestion
_Thanks to the following people for helping with this post: Olaf Geirsson, Rebecca Dodd, Thorsten Ball, Erica Lindberg, Malo Marrec, Victoria Yunger, Beyang Liu. We welcome [edits to this post](https://github.com/sourcegraph/about/tree/main/blogposts/2021/log4j-log4-shell-0day.md) by pull request._
```

**Pull request merged**